thaomonster/ACNH-Collections
# Feature/basic structure setup

### What does this PR do? 
>Clearly and concisely describe the feature.
 
This PR builds out the structure for the react app.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

None.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?

There were a lot of files changes some were taken out and some were added in.

### How should this be tested?
>What steps need to be taken to confirm functionality? 

Open the codebase and look at the file structure.

### Steps To Take
>Any questions and possible steps moving forward.

None.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #1
